src/axom/CMakeLists.txt

```diff
@@ -106,3 +106,25 @@ if(WIN32)
             TARGET_DEFINITIONS "axom_EXPORTS")
     endforeach()
 endif()
+
+
+#------------------------------------------------------------------------------
+# With C++11, some Axom types in src/axom/core/Types.hpp are obsolete.
+# They will be removed in steps, as the AXOM_DEPRECATED_TYPES variable
+# defaults to WARN, then ERROR, then eventually removed.
+#------------------------------------------------------------------------------
+if (NOT AXOM_DEPRECATED_TYPES)
+  set( AXOM_DEPRECATED_TYPES "WARN" )
+endif()
```

Comment:
Please move the `AXOM_DEPRECATED_TYPES` option to `AxomOptions.cmake` and add a `STRINGS` property, as we do with `AXOM_DEBUG`.

The logic associated with checking validity might be better in `AxomConfig.cmake`.

See: https://github.com/LLNL/axom/blob/develop/src/cmake/AxomOptions.cmake

Reply:
Done.